packages/Chem/src/descriptors/descriptors-calculation.ts

```diff
@@ -14,12 +15,19 @@ export async function addDescriptors(smilesCol: DG.Column, viewTable: DG.DataFra
   openDescriptorsDialog(await getSelected(), async (selected: any) => {
```

Comment:
It should be pure function that aceepts column and returns columns, optionally providing progress information to caller